ts/screens/wallet/TransactionsScreen.tsx

```diff
@@ -76,14 +83,41 @@ const styles = StyleSheet.create({
 
   brandDarkGray: {
     color: variables.brandDarkGray
+  },
+
+  cardContainer: {
+    height: 235,
+    width: "100%",
+    position: "absolute",
+    top: 16,
+    zIndex: 7,
+    elevation: 7,
+    alignItems: "center"
+  },
+  headerSpacer: {
+    height: 172
+  },
+  cancelButton: {
+    borderColor: IOColors.red,
+    width: '90%',
+    height: variables.btnHeight + 10,
+    marginHorizontal: variables.contentPadding,
+    marginTop: 16,
   }
 });
 
+
 const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
   title: "wallet.walletCardTransaction.contextualHelpTitle",
   body: "wallet.walletCardTransaction.contextualHelpContent"
 };
 
+const UnsubscribeButton = (props: { onPress?: () => void }) => (
+  <Button bordered={true} style={styles.cancelButton} onPress={props.onPress}>
```

Comment:
We are trying to use only IO components
So in that way we prefer the usage of ButtonDefaultOpacity that fills the IO design and behavior
see https://github.com/pagopa/io-app/blob/d3b11ca022bcf3f1f9c15db60aa40812bca72295/ts/features/bonus/bpd/screens/details/components/unsubscribe/UnsubscribeToBpd.tsx#L46